### ISSUE TITLE
Path not found error when installing EA app

### ISSUE THREAD

**Issue by vgzotta** [moraroy/NonSteamLaunchers-On-Steam-Deck#93]
Hello,

Tried installing a few apps. All of them installed fine first time and all of them are working fine (logged in already), but the EA app did not launch in game mode (the window was empty not matter what proton version I tried). So I thought to uninstall it and reinstall. 

I uninstalled but then it won't install at all. It was blocked at downloading. I changed the WiFi to a different one and it now works, but it says Path not found. I click OK, then it says installation complete and restarts Steam. But there is no EA app showing in Steam. I also checked program files in c drive. There is no EA app installed there. 

I have no idea what to do next. I can try looking for a tutorial to install it manually, but I just got my Deck and I'm still learning things. This is why Non Steam Launchers was really helpful. I also attached the log as I think it could help. Thank you!
[NonSteamLaunchers-install.log](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/files/12070201/NonSteamLaunchers-install.log)

**Comment by callit** [comment]
I believe that this relates to adding a new launcher while using the same prefix @moraroy.  It seems that proton builds out the empty prefix folder when the app is first installed, but it doesn't check whether new files are required when subsequent apps are run with that same prefix. That's my take on what's happening anyway.

This would also possibly explain the issue I thought I saw with missing files on the Humble launcher and why the "start clean" option resolved those issues.

Line 345 from the install log: Unable to read VR Path Registry from C:\users\steamuser\AppData\Local\openvr\openvrpaths.vrpath

I went through the script and found the EA app install code.  When I used the NonSteamLaunchers prefix, I got this same error (I already have Humble and only Humble installed).  However, when I ran the installer using the separate prefix logic and used TheEAappLauncher, the prefix folder was populated and had the expected file in it. 

To anyone experiencing this, using the top check box to "SEPARATE APP IDS" may help.

Unfortunately, this may all be moot because I think the EA launcher is broken again with Proton.  After the install, I just get a blank grey box.  I'm going to try one other thing to get it working, I'll report back if there's good news.
